lib/main.js

```diff
@@ -2,6 +2,7 @@ const glob = require('glob')
 const fs = require('fs')
 
 const rss = require('./rss')
+const pre2017 = require('./pre2017')
```

Comment:
Give a blank line after this `const` please.

Reply:
coala give a patch to remove line break

Reply:
Then patch it before push a commit.